addons/payment/tests/test_flows.py

```diff
@@ -1,6 +1,8 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 from freezegun import freeze_time
+import json
+import unittest.mock
```

Comment:
That was a suggestion from JUC, to be sure it's not the patch method from Odoo, it may be longer, but cleaner.

So, later on, I still use `unittest.mock.patch` instead of `patch`

Reply:
Ok then, let's keep it like that :ok_hand:

Reply:
As discussed with anv, we don't use the patch method from Odoo, so this won't cause a problem, I go back to `from unittest.mock import patch` ^^